client/src/containers/edit/index.js

```diff
@@ -0,0 +1,60 @@
+/* This file is the main edit page, which is
+* the page that users will see as soon as they click the
+* edit page from the menu*/
+import React, { Component } from 'react'
+import
+{ Paper,
+  Menu,
+  MenuItem,
+  Divider,
+  AppBar,
+  IconButton
+} from 'material-ui'
+import './main.css'
+import Person from 'material-ui/svg-icons/social/person'
+import School from 'material-ui/svg-icons/social/school'
+import Professional from 'material-ui/svg-icons/action/work'
+import Home from 'material-ui/svg-icons/action/home'
+import EditPersonal from './EditPersonal'
```

Comment:
Lines 18 and 19 importing 'EditPersonal' and 'EditProfessional' are throwing linting errors, consider commenting out until needed.